content/docs/1.7.0/high-availability/rwx-volume-fast-failover.md

```diff
@@ -20,3 +20,27 @@ For more information, see https://github.com/longhorn/longhorn/issues/6205.
 > kubectl -n longhorn-system delete lease pvc-2ce4e82e-7ccc-46c0-90a8-a141501fbf93
 > ```
 > See, for example, https://github.com/longhorn/longhorn/issues/9093.
+
+### Resource Consumption and System Performance Impact
+
+The Longhorn team has investigated the impact of RWX volumes on resource consumption and system performance. The benchmarking studies, which were completed using 60 RWX volumes, show that enabling the *RWX volume fast failover feature* results in the following: 
+
+- Increased API request rate
+- Higher RPC rates 
+- Slight increase in CPU and memory usage
+
+#### **Environment:**
+- **Setup:** 1 Control Node + 3 Worker Nodes (v1.27.15+rke2r1)
+- **Workload:** 60 Deployments with 60 RWX volumes with `soft` mount
+
+#### **Test Results:**
+
+| **Metric**                           | **Fast Failover Enabled** | **Fast Failover Disabled** | **Difference**             |
+|--------------------------------------|---------------------------|----------------------------|----------------------------|
+| **1. Number of API Requests**        | 59 req/s                  | 37.5 req/s                 | **+57.3%**                 |
+| **2. RPC Rate**                      | 57 ops/s                  | 37 ops/s                   | **+54.1%**                 |
+| **3. Memory Usage**                  | Higher Peaks/Minima       | Lower Peaks/Minima         | More usage with Fast Failover Enabled |
```

Comment:
What target is for these metrics? Kubernetes API Server?  What's RPC here?

Reply:
Hi @innobead 
1. `Number of API Requests` is number of requests to Kube API server
1. `RPC rate` is the number of RPC calls from Kube API server to ETCD. API server is talking to ETCD via RPC calls. This can be used to indicate the load on ETCD

The target for these metrics is that when enabling RWX fas failover, they should not increase by more than 60% of the base line (when disable RWX fast failover)

Reply:
Thank you. Very clear.